### PR TITLE
fix(upload-local-provider): use package require instead of absolute path

### PR DESCRIPTION
**Pull request by nathan-pichon**
### What does it do?

Replace require from an absolute path to package require.

### Why is it needed?

As not part of the same package, installed independently provider upload local will fail to require the file utils.

### CODE REVIEW

**Review by Convly — COMMENTED** [review]
Nice idea, I was thinking that maybe it could be better to let the ownership of the file utils to the upload plugin? (through the use of a service). What do you think?

Also, if we want to proceed with the file utils in the strapi-utils packages, could you update the main `strapi/utils` export & all the imports so that they target `strapi/utils` instead of `strapi/utils/lib/file`?

**Comment by Convly** [comment]
> Nice idea, I was thinking that maybe it could be better to let the ownership of the file utils to the upload plugin? (through the use of a service). What do you think?

Basically, it would mean having

`strapi.plugin('upload').service('utils').file` to which you have access whenever you have a strapi instance

**Comment by nathan-pichon** [comment]
> 

As this file utils is now used by several packages, I think it has its place among other utils in `@strapi/utils`. 🤔 
I'm not a huge fan of getting everything through the services of Strapi. It obfuscates a lot of where the functions are stored and how to access/review them. 😅 
I can still do both 🤔 
Would we prefer to stick with the service architecture and have a "kinda god object" Strapi over the requires of "external" packages?

Will do the update for `@strapi/utils` export/import. 👍  It was planned, but wanna go fast on this. 🙂

**Comment by Convly** [comment]
Let's stick with what you've done here, if you could just update the export/requires that would be perfect

**Comment by nathan-pichon** [comment]
> Let's stick with what you've done here, if you could just update the export/requires that would be perfect

@Convly Done 👍

**Review by Convly — CHANGES_REQUESTED** [review]

File: packages/core/upload/server/services/image-manipulation.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ const fs = require('fs');
 const { join } = require('path');
 const sharp = require('sharp');
 
+const { bytesToKbytes, writableDiscardStream } = require('@strapi/utils/lib/file');
```
[the remark this answers is not in the document]
Comment (nathan-pichon) [reply packages/core/upload/server/services/image-manipulation.js:10]: Oupsy

[the remark this answers is not in the document]
Comment (nathan-pichon) [reply packages/core/upload/server/services/image-manipulation.js]: Should be good now 🆗

**Review by Marc-Roig — APPROVED** [review]

**Review by Convly — APPROVED** [review]

**Pull request merged**